include/core/boundary_conditions.h

```diff
@@ -35,6 +35,8 @@ namespace BoundaryConditions
 {
   enum class BoundaryType
   {
+    // common
+    none,
```

Comment:
```suggestion
    zero_flux,
```

Reply:
I use `none` for the VOF BC, so I will not remove it. I will just create an additional `noflux` :)